docs/OnTrack/Voice Verification/Architecture-Document.md

```diff
@@ -0,0 +1,52 @@
+# Architecture Document | Voice Verification for OnTrack Delivery
+## Introduction
+### Purpose
+This document provides a comprehensive architectural overview of the system, using a few different architectural views to depict different aspects of the system. It is intended to capture and convey the significant architectural decisions which have been made on the system. 
```

Comment:
The purpose and scope are basically the same paragraph. Perhaps in the scope you can outline the different aspects of the system that a being depicted eg. "This document will address the front end, back end and decisions making but will not address how to use the system" or something like that.